build.gradle.kts

```diff
@@ -4,7 +4,9 @@ plugins {
 
 allprojects {
     extra["groupId"] = "dev.openfeature.kotlin.contrib"
+// x-release-please-start-version
     ext["version"] = "0.1.0"
```

Comment:
Can the versions be managed by each artifact? I've updated Release Please to support monorepos.